test/service/test_response.py

```diff
@@ -2035,24 +2041,45 @@ def test_summary_filter_none(self):
 
 class TestUnpopulatedIndexResponse(WebServiceTestCase):
 
+    @classmethod
+    def bundles(cls) -> List[BundleFQID]:
+        return []
+
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.index_service.create_indices(cls.catalog)
+        cls._setup_indices()
 
     @classmethod
     def tearDownClass(cls):
-        cls.index_service.delete_indices(cls.catalog)
+        cls._teardown_indices()
         super().tearDownClass()
 
+    entity_types = ['files', 'projects', 'samples', 'bundles']
+
     def test_empty_response(self):
-        url = self.base_url + "/index/projects"
-        response = requests.get(url)
-        response.raise_for_status()
-        response = response.json()
-        self.assertEqual([], response['hits'])
-        self.assertEqual({None}, set(response['pagination'].values()))
-        self.assertEqual({}, response['termFacets'])
+        for entity_type in self.entity_types:
+            with self.subTest(entity_type=entity_type):
+                response = requests.get(url=furl(url=self.base_url,
+                                                 path=f'/index/{entity_type}').url)
+                response.raise_for_status()
+                response = response.json()
+                self.assertEqual([], response['hits'])
+                self.assertEqual(0, response['pagination']['count'])
+                self.assertEqual(0, response['pagination']['total'])
+                self.assertTrue(response['termFacets'])
+                for value in response['termFacets'].values():
```

Comment:
Looks like you didn't address https://github.com/DataBiosphere/azul/pull/2396#discussion_r521193167

Reply:
I thought adding`self.assertTrue(response['termFacets'])` was sufficient to ensure that there would be values present. Since we removed the special response case I thought the `termFacets` would always be populated...

Reply:
I missed the addition of assertTrue. Yes, that would prevent an empty `response['termFacets']` from passing the test but is there a reason why this can be a simple `assertEquals(…, response['termFacets'])`?